src/BoundaryConditions/BoundaryConditions.jl

```diff
@@ -14,7 +14,7 @@ using CUDA
 using KernelAbstractions: @index, @kernel
 
 using Oceananigans.Architectures: CPU, GPU, device
-using Oceananigans.Utils: work_layout, launch!
+using Oceananigans.Utils: work_layout, launch!, KernelParameters
```

Comment:
let's move the
`using Oceananigans.Utils: KernelParameters` to 
[fill_halo_regions.jl](https://github.com/CliMA/Oceananigans.jl/pull/3125/files#diff-e6c8318633fd62c744bd0d83e9145622587263d81359648828794640d09d3766)